src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -23,6 +23,26 @@
 JOB_ID_LIST = "job_id_list"
 CELL_ID_LIST = "cell_id_list"
 
+# messages
```

Comment:
some nice constants in place of the message type names

Reply:
all changes in this file are either search-replace or minor documentation changes